test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -440,6 +440,28 @@ public void ApplyFunctionsFilter()
         }
 
         [Fact]
+        public void ApplyTimeout()
```

Comment:
nit: I'd rename to ApplyConfiguration_AppliesFunctionTimeout. I realize the test above was poorly named - rename that one to ApplyConfiguration_AppliesFunctionsFilter as well :)